source/ed/connectors/gtfs_parser.cpp

```diff
@@ -501,12 +517,91 @@ StopsGtfsHandler::stop_point_and_area StopsGtfsHandler::handle_line(Data& data,
             return_wrapper.first = sp;
         }
         return return_wrapper;
+        // Handle I/O case
+    } else if (has_col(type_c, row) && row[type_c] == "3") {
+        auto* io = build_input_output(data, row);
+        if (io) {
+            return {};
+        }
+        return return_wrapper;
     } else {
         // we ignore pathways nodes
         return {};
     }
 }
 
+void PathWayGtfsHandler::init(Data&) {
+    pathway_id_c = csv.get_pos_col("pathway_id");
+    from_stop_id_c = csv.get_pos_col("from_stop_id");
+    to_stop_id_c = csv.get_pos_col("to_stop_id");
+    pathway_mode_c = csv.get_pos_col("pathway_mode");
+    is_bidirectional_c = csv.get_pos_col("is_bidirectional");
+    length_c = csv.get_pos_col("length");
+    traversal_time_c = csv.get_pos_col("traversal_time");
+    stair_count_c = csv.get_pos_col("stair_count");
+    max_slope_c = csv.get_pos_col("max_slope");
+    min_width_c = csv.get_pos_col("min_width");
+    signposted_as_c = csv.get_pos_col("signposted_as");
+    reversed_signposted_as_c = csv.get_pos_col("reversed_signposted_as");
+}
+
+int PathWayGtfsHandler::fill_pathway_field(const csv_row& row, const int key, const std::string type) {
+    int pw_value = UNKNOWN_COLUMN;
```

Comment:
The constant UNKNOWN_COLUMN is used in the rest of the code to handle a column name which is absent in the csv, but expected to be there by the code.
Here you use the same constant to signal that some value in the csv is ill-formed (a float was expected).
Maybe use another constant ?

Reply:
probably a `DEFAULT_PATHWAYS_VALUE` is better yes